frontend/src/bundles/profile/components/profile-settings/profile-settings.tsx

```diff
@@ -0,0 +1,229 @@
+import {
+    Avatar,
+    Button,
+    ButtonVariant,
+    DatePicker,
+    Input,
+    Loader,
+    Radio,
+} from '~/bundles/common/components/components.js';
+import { IconColor } from '~/bundles/common/components/icon/enums/enums.js';
+import { ComponentSize, Gender } from '~/bundles/common/enums/enums.js';
+import {
+    configureDateString,
+    configureISOString,
+} from '~/bundles/common/helpers/helpers.js';
+import {
+    useAppForm,
+    useAppSelector,
+    useCallback,
+    useEffect,
+    useState,
+} from '~/bundles/common/hooks/hooks.js';
+import {
+    type UserUpdateProfileRequestDto,
+    userUpdateProfileValidationSchema,
+} from '~/bundles/users/users.js';
+
+import { DEFAULT_UPDATE_PROFILE_PAYLOAD } from './constants/constants.js';
+
+type Properties = {
+    onSubmit: (payload: UserUpdateProfileRequestDto) => void;
+    isLoading: boolean;
+};
+
+const ProfileSettings: React.FC<Properties> = ({ onSubmit, isLoading }) => {
+    const [valuesDefault, setValuesDefault] = useState(false);
+    const { user } = useAppSelector(({ auth }) => ({
+        user: auth.user,
+    }));
+
+    const { control, errors, reset, getValues, setValue } =
```

Comment:
Is it impossible to use handleSubmit from hook form?

Reply:
I think I can)